src/models/modules/CourtConfigManager.js

```diff
@@ -18,11 +18,11 @@ module.exports = class extends BaseDeployer {
   }
 
   async setConfigDirectly() {
-    const { court, jurors } = this.config
+    const { modules: { court: courtAddress }, court, jurors } = this.config
     logger.info('Setting Court config...')
-    const fromTermId = (await this._getCurrentTerm()).add(1)
+    const fromTermId = (await this._getCurrentTerm()).add(bn(1))
     const AragonCourt = await this.environment.getArtifact('AragonCourt', '@aragon/court')
-    const controller = await AragonCourt.at(court)
+    const controller = await AragonCourt.at(courtAddress)
```

Comment:
😅 I guess this part has been broken for a while, probably due to model changes a few months back?

Reply:
Yes, although this case was being used for test cases only (local deployments)